checkstyle-config/pom.xml

```diff
@@ -57,7 +57,7 @@ limitations under the License.
     <url>https://github.com/GoogleCloudPlatform/java-repo-tools</url>
     <connection>scm:git:https://github.com/GoogleCloudPlatform/java-repo-tools.git</connection>
     <developerConnection>scm:git:ssh://git@github.com/GoogleCloudPlatform/java-repo-tools.git</developerConnection>
-    <tag>v1.0.3-alpha</tag>
+    <tag>v1.0.1</tag>
```

Comment:
LOL - yep - the 1.0.1 would have been the official number.  Of course, I ran into trouble uploading, on Saturday, and didn't get back to it.